dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -27,7 +27,7 @@ HRESULT InitializePlatform()
 {
     std::call_once(m_isPlatformInitialized,
         [&] {
-            g_platform = Microsoft::WRL::Make<WpnLrpPlatformImpl>();
+            g_platform = Microsoft::WRL::Make<NotificationsLongRunningPlatformImpl>();
```

Comment:
Why global? I thought the whole point of ownership under factory class is that this could be a member variable of the factory class

Reply:
Please remove global platform

Reply:
The latest iteration just rebased against main. The relevant changes addressing ownership will come later.